src/p_enemy.c

```diff
@@ -8831,7 +8820,41 @@ void A_ChangeColorAbsolute(mobj_t *actor)
 			actor->color = actor->target->color;
 	}
 	else
-		actor->color = (UINT8)locvar2;
+		actor->color = (UINT16)locvar2;
+}
+
+// Function: A_Dye
+//
+// Description: Colorizes an object.
+//
+// var1 = if (var1 != 0), dye your target instead of yourself
+// var2 = color value to dye
+//
+void A_Dye(mobj_t *actor)
```

Comment:
```
p_enemy.c:8867:6: error: redefinition of 'A_Dye'
 void A_Dye(mobj_t *actor)
      ^~~~~
p_enemy.c:8833:6: note: previous definition of 'A_Dye' was here
 void A_Dye(mobj_t *actor)
      ^~~~~
p_enemy.c: In function 'A_Dye':
p_enemy.c:8876:15: error: 'MAXTRANSLATIONS' undeclared (first use in this function)
  if (color >= MAXTRANSLATIONS)
               ^~~~~~~~~~~~~~~
p_enemy.c:8876:15: note: each undeclared identifier is reported only once for each function it appears in
```

Reply:
I really should not have tried to merge the next-lc-uint16 branch into next-luacolors. I'll close this PR and make a new one with the new branch.